python/example_code/bedrock-agent-runtime/bedrock_agent_runtime_wrapper.py

```diff
@@ -68,4 +68,60 @@ def invoke_agent(self, agent_id, agent_alias_id, session_id, prompt):
 
     # snippet-end:[python.example_code.bedrock-agent-runtime.InvokeAgent]
 
-# snippet-end:[python.example_code.bedrock-agent-runtime.BedrockAgentsRuntimeWrapper.class]
+        # snippet-start:[python.example_code.bedrock-agent-runtime.InvokeFlow]
+    def invoke_flow(self, flow_id, flow_alias_id, input_data, execution_id):
+        """
+        Invoke an Amazon Bedrock flow and handle the response stream.
+
+        Args:
+            client: Boto3 client for Amazon Bedrock agent runtime.
```

Comment:
I don't see an arg named `client`, maybe it was removed?

Reply:
Typo. I removed it.